### DISCUSSION TITLE
Rift objectives count doesn't match the count from SkyBlock level

### DISCUSSION THREAD

**Discussion by MattTheCuber** [HypixelDev/PublicAPI#640] [Q&A] [answered]
There are currently 98 rift objectives listed in the API objectives but maximum amount for SkyBlock experience is 97 objectives (291 / 3). This means 1 of 2 things. Hypixel has a bug (either in the API or in leveling) or this is intentional and one of the objectives should not be counted towards SkyBlock experience. If anyone has tested if it is this second option and knows what quest it is, please share.

![image](https://github.com/HypixelDev/PublicAPI/assets/32849887/48085d43-18b9-44f9-8cff-22005966a375)

**Comment by ConnorLinfoot** [comment, marked as answer]
Looks like the objective `rift_get_inside_necklace` is the one that is not considered part of the rift for what the XP task uses, I'll check internally if there is any knowledge if that is intentional or not, may have an answer next week with it being the weekend now.

**Reply by MattTheCuber** [reply]
Awesome, so many problems can just be solved by asking! Thank you again!

**Reply by ConnorLinfoot** [reply]
I've had confirmation that it is intended that this objective is excluded from the Rift XP task, if my understanding is correct this is because the necklace objective leads to another XP task being completed in another form (or something along those lines)

**Reply by MattTheCuber** [reply]
Great, thank you for looking into this!
